README.md

```diff
@@ -13,6 +13,8 @@ They feature advanced functionality for data-rich applications and a wide range
 MUI X is **open core**—base components are MIT-licensed, while more advanced features require a Pro or Premium commercial license.
 See the [Licensing page](https://mui.com/x/introduction/licensing/) for details.
 
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/6293/badge)](https://bestpractices.coreinfrastructure.org/projects/6293)
```

Comment:
🙌 
What do you think about putting this badge just after the image/name, before any text - after line 6 in our case.
From what I can see, it's a more common approach by other libraries, like the one you linked or:
- [React](https://github.com/facebook/react)
- [Vite](https://github.com/vitejs/vite)
- [freeCodeCamp](https://github.com/freeCodeCamp/freeCodeCamp)
- [Flutter](https://github.com/flutter/flutter)

FYI, have you considered adding [scorecard generation](https://securityscorecards.dev/) as well? 🤔

Reply:
I have copied https://github.com/mui/material-ui#readme when it comes to where to place the badges, no real preferences.

> FYI, have you considered adding [scorecard generation](https://securityscorecards.dev/) as well?

Yeah, it could make sense, for the badge as well.